Sources/S3Signer/Region.swift

```diff
@@ -4,68 +4,68 @@ import Foundation
 /// AWS Region
 public struct Region {
     
-    /// name of the region, see RegionName
-    public let name: RegionName
+    /// name of the region, see Name
+    public let name: String
```

Comment:
this introduces a breaking change that won't bring much benefit to most users, could you please keep the original enum (it was Codable so you can store info about easily in a db for example) and just add `.custom(String)` as a case? I thought that's what was suggested on discord?